src/openMVG/calibration/calibration.hpp

```diff
@@ -9,13 +9,13 @@ namespace calibration{
 /**
  * @brief This function computes the average of the reprojection errors.
  *
- * @param[in] objectPoints
- * @param[in] imagePoints
- * @param[in] rvecs
- * @param[in] tvecs
- * @param[in] cameraMatrix
- * @param[in] distCoeffs
- * @param[out] perViewErrors
+ * @param[in] objectPoints Coordinates of the 3D points in each image of the sequence.
+ * @param[in] imagePoints Coordinates of the 2D points in each image of the sequence.
+ * @param[in] rvecs Rotation of the camera of each image.
+ * @param[in] tvecs Position of the camera of each image.
+ * @param[in] cameraMatrix The camera parameters.
+ * @param[in] distCoeffs The distortion coefficients.
+ * @param[out] perViewErrors The reprojection errors for each image.
  * @return The average of the reprojection errors.
  */
 double computeReprojectionErrors(const std::vector<std::vector<cv::Point3f> >& objectPoints,
```

Comment:
Ok i see, for cctags, since we have ID, we can detect only a partial part of the pattern in the image (which does not work for opencv methods with checkerboard and circles).